### PR TITLE
fix: update with new sdk for item login changes

### PR DESCRIPTION
**Pull request by spaenleh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
   "devDependencies": {
     "@commitlint/cli": "19.3.0",
     "@commitlint/config-conventional": "19.2.2",
-    "@graasp/sdk": "4.21.0",
+    "@graasp/sdk": "github:graasp/graasp-sdk#568-new-membertype",
```
Comment (ReidyT) [inline package.json:34]: reminder

File: src/mutations/membership.ts [hunk 2]
```diff
@@ -198,10 +198,12 @@ export default (queryConfig: QueryClientConfig) => {
...
-            // set map key to email
-            Object.values(membershipsResult.data).forEach((m) => {
-              dataForMemberships.data[m.member.email] = m;
-            });
+            // FIX: this is commented out as we will not receive the email from the backend anymore.
+            // we should remove this code anyways and move the whole process to the backend.
+            // // set map key to email
+            // Object.values(membershipsResult.data).forEach((m) => {
+            //   dataForMemberships.data[m.account.email] = m;
+            // });
```
Comment (ReidyT) [inline src/mutations/membership.ts:206]: Should we create and issue and remove the comment for that ?

**Review by ReidyT — APPROVED** [review]
LGTM ☄️

File: src/mutations/membership.ts (hunk 2 above)
Comment (spaenleh) [reply src/mutations/membership.ts:206]: I think @pyphilia is working on adding the necessary code in the backend ?

Comment (pyphilia) [reply src/mutations/membership.ts:206]: I'm going to.

**Review by pyphilia — APPROVED** [review]

**Pull request merged**